packages/backend/src/open_payments/payment/outgoing/model.ts

```diff
@@ -144,7 +146,7 @@ export class OutgoingPayment
     return {
       id: json.id,
       accountId: json.accountId,
-      state: json.state.toLowerCase(),
+      state: json.state,
```

Comment:
Why do we not need the `toLowerCase()`?

Reply:
Oh, because this if what the GraphQL resolver uses, right?

Reply:
It's more that the only thing that wanted lower case states was Open Payments, but states are no longer returned in any of the Open Payments (sub)resources